prac_05/word_occurrences.py

```diff
@@ -0,0 +1,26 @@
+"""
+Count word occurrences in a string, sort alphabetically case-insensitive.
+
+Estimate: < 1 hour
+Actual:   1.5 hours
+"""
+
+word_occurrences = {}
+
+string = input("Enter text: ").lower()
+string = list(string.split())
```

Comment:
Using string for both the text and the list version can be a bit confusing. Since this is a list of words, naming it words might be clearer and more descriptive.